examples/stateful_s3_example.yaml

```
@@ -0,0 +1,177 @@
+        Authorization: "Bearer ${ALERT_API_KEY}"
+        Content-Type: "application/json"
+    error_output:
+      type: elasticsearch
+      hosts: ["https://elasticsearch.example.com:9200"]
+      index: "arkflow-errors"
+      username: "${ES_USERNAME}"
+      password: "${ES_PASSWORD}"
+    state:
+      operator_id: "iot-monitor"
+      enabled: true
+      state_timeout_ms: 604800000  # 7 days
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)